Description: Fix hypens
 Minus symbols, used for command flags, are interpreted by roff as
 hypens unless escaped.
 .
 This patch escapes a bunch of minus symbols.
Author: Andrea Bolognani <eof@kiyuko.org>
Last-Update: 2011-09-14
Index: scrotwm.git/scrotwm.1
===================================================================
--- scrotwm.git.orig/scrotwm.1	2011-09-14 18:54:45.219453391 +0200
+++ scrotwm.git/scrotwm.1	2011-09-14 19:25:35.456511954 +0200
@@ -659,22 +659,22 @@
 property of the root window. This can be used for example to retrieve the 
 title of the currently active window with xprop(1) and grep(1):
 .Bd -literal -offset indent
-$ WINDOWID=`xprop -root _NET_ACTIVE_WINDOW | grep -o "0x.*"`
-$ xprop -id $WINDOWID WM_NAME | grep -o "\\".*\\""
+$ WINDOWID=`xprop \-root _NET_ACTIVE_WINDOW | grep \-o "0x.*"`
+$ xprop \-id $WINDOWID WM_NAME | grep \-o "\\".*\\""
 .Ed
 .Pp
 A window can be focused by sending a _NET_ACTIVE_WINDOW client message 
 to the root window. For example, using wmctrl to send the message 
 (assuming 0x4a0000b is the id of the window to be focused):
 .Bd -literal -offset indent
-$ wmctrl -i -a 0x4a0000b
+$ wmctrl \-i \-a 0x4a0000b
 .Ed
 .Pp
 Windows can be closed by sending a _NET_CLOSE_WINDOW client message 
 to the root window. For example, using wmctrl to send the message 
 (assuming 0x4a0000b is the id of the window to be closed):
 .Bd -literal -offset indent
-$ wmctrl -i -c 0x4a0000b
+$ wmctrl \-i \-c 0x4a0000b
 .Ed
 .Pp
 Windows can be floated and un-floated by adding or removing the 
@@ -684,7 +684,7 @@
 a window using wmctrl to send the message (assuming 0x4a0000b is the 
 id of the window floated or un-floated):
 .Bd -literal -offset indent
-$ wmctrl -i -r 0x4a0000b -b toggle,_NET_WM_STATE_ABOVE
+$ wmctrl \-i \-r 0x4a0000b \-b toggle,_NET_WM_STATE_ABOVE
 .Ed
 .Pp
 Floating windows can also be resized and moved by sending a 
@@ -692,7 +692,7 @@
 using wmctrl to send the message (assuming 0x4a0000b is the id of 
 the window to be resize/moved):
 .Bd -literal -offset indent
-$ wmctrl -i -r 0x4a0000b -e 0,100,50,640,480
+$ wmctrl \-i \-r 0x4a0000b \-e 0,100,50,640,480
 .Ed
 .Pp
 This moves the window to (100,50) and resizes it to 640x480.
